### PR TITLE
New layout

### PR DESCRIPTION
**Pull request by tmarkn**
with merged label-creation

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -1,55 +1,189 @@
...
+            <div class="highlight-color color-selected" style="background-color: red" id="red">
+              <p></p>
+            </div>
+            <div class="highlight-color" style="background-color: orange" id="orange">
             </div>
+            <div class="highlight-color" style="background-color: yellow" id="yellow">
+            </div>
+            <div class="highlight-color" style="background-color: green" id="green">
+            </div>
+            <div class="highlight-color" style="background-color: blue" id="blue">
```
Comment (ghost) [inline templates/index.html:95]: I see the colors are defined but they are not showing up in my browser

File: static/js/app.js [hunk 2]
```diff
@@ -5,21 +5,23 @@ var testContent = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis
 
 var textArea = document.querySelector('#doc_view');
 var currentDocument = new Doc("test", testContent);
-var currentHighlighterColor;
-// future implementation
+var currentHighlighterColor = $('.color-selected')[0].id;
```
Comment (ghost) [inline static/js/app.js]: Referencing a variable by array index should generally be a last resort. Makes code hard to read and isn't descriptive.

File: static/js/app.js [hunk 3]
```diff
@@ -5,21 +5,23 @@ var testContent = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis
...
+// //future implementation
 var openDocuments = [];
-var currentLabel;
+var currentLabel = $('.label-selected')[0].getAttribute('value');
```
Comment (ghost) [inline static/js/app.js]: samesies here

File: static/js/app.js [hunk 4]
```diff
@@ -93,16 +95,30 @@ renderTextareaHighlights = function () {
...
 
 //change the document's label context
-$('input[type=radio]').change(function () {
-    currentLabel = this.value;
+$('.label').on('click', function (e) {
+  console.log(e.currentTarget.getAttribute('value'));
+  currentLabel = event.currentTarget.getAttribute('value');
+  $('.label').removeClass('label-selected');
+  $(e.currentTarget).addClass('label-selected');
 });
```
Comment (ghost) [inline static/js/app.js:103]: this is not working for me

File: templates/index.html [hunk 5]
```diff
@@ -1,55 +1,189 @@
...
+              </div>
+            </div>
+            <hr>
+            </hr>
+            <div class="list-group">
+              <h4>Label 2</h4>
+              <div class="ml-3">
+                <a href="" class="annotation">word 1</a><br>
+                <a href="" class="annotation">word 2</a><br>
+                <a href="" class="annotation">word 3</a><br>
```
Comment (ghost) [inline templates/index.html:133]: These shouldn't be clickable

File: templates/index.html [hunk 6]
```diff
@@ -1,55 +1,189 @@
...
+            <hr>
+            </hr>
+            <div class="list-group">
+              <h4>Label 4</h4>
+              <div class="ml-3">
+                <a href="" class="annotation">word 1</a><br>
+                <a href="" class="annotation">word 2</a><br>
+                <a href="" class="annotation">word 3</a><br>
+                <a href="" class="annotation">word 4</a><br>
+                <a href="" class="annotation">word 5</a><br>
```
Comment (ghost) [inline templates/index.html:153]: same here

**Review by ghost — CHANGES_REQUESTED** [review]
Site looks great but some fairly problematic things I noticed:
* the website is cut off at the top
* colors aren't listed
* can't select labels
* textarea size changes when clicked

File: templates/index.html (hunk 5 above)
Comment (tmarkn) [reply templates/index.html:133]: I was assuming we were adding a way to click the annotation and scroll to it

Comment (ghost) [reply templates/index.html:133]: Gotcha. If that's what we're doing then this is resolved

File: templates/index.html (hunk 1 above)
Comment (tmarkn) [reply templates/index.html:95]: Which browser are you using? I checked chrome, opera, firefox, and edge

**Review by tuj52136 — DISMISSED** [review]
looks good!

File: static/js/app.js (hunk 2 above)
Comment (tmarkn) [reply static/js/app.js]: removed reference by array index

File: static/js/app.js (hunk 3 above)
Comment (tmarkn) [reply static/js/app.js]: removed reference by array index here too

**Review by ghost — APPROVED** [review]

**Review by tuj52136 — APPROVED** [review]

**Pull request merged**